src/test/java/com/conveyal/gtfs/util/PolylineUtilsTest.java

```diff
@@ -0,0 +1,109 @@
+package com.conveyal.gtfs.util;
+
+import org.junit.Test;
+import org.locationtech.jts.geom.Coordinate;
+import org.locationtech.jts.geom.GeometryFactory;
+import org.locationtech.jts.geom.Point;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static com.conveyal.gtfs.util.PolylineUtils.decode;
+import static com.conveyal.gtfs.util.PolylineUtils.encode;
+import static com.conveyal.gtfs.util.PolylineUtils.simplify;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+public class PolylineUtilsTest {
```

Comment:
Is this test case also taken from mapbox-java? If so we should properly attribute.